Server/src/controllers/getDatasetController.ts

```diff
@@ -50,49 +50,95 @@ export class getDataController {
      */
     async createRequestForUserUploadedDatasets(request: Request, response: Response) {
         let requestParam = request.params.userUploadedDatasets;
-        let userId: number = +requestParam;
-        if (isNaN(userId)) {
-            response.status(400).send("Invalid search params entered");
+        try {
+            const retrieveDataObject = new retrieveData();
+            let executionStatus = await retrieveDataObject.getUserUploadedDatasets(requestParam)
+            if (executionStatus[0]) { return response.status(200).json(executionStatus[1]); }
+            else { return response.status(400).json(executionStatus[1]); }

```

Comment:
Will this always be true? If ExecutionStatus[1], does this always mean its a bad Request